### ISSUE TITLE
Prevent slider from swapping handles

### ISSUE THREAD

**Issue by ambrt** [fomantic/Fomantic-UI#1021]
# Feature Request

Slider should have option to  block "max handle" when trying to go below "min handle"

## Description
The slider handles can be reversed, so if i drag max handle to the left of min handle, values get inverted.

There should be option in settings to prevent this.

## Example
![ui-slider-bug](https://user-images.githubusercontent.com/10456131/65122874-4b4a8a00-d9f2-11e9-90e8-ec2c6d9f2bd8.gif)

**Comment by ambrt** [comment]
In examples this feature is achieved by
```
 if (firstVal > secondVal) {
          $('#range-slider-input-1').val('|' + firstVal + " - " + secondVal + '| = ' + range);
        } else {
          $('#range-slider-input-1').val('|' + secondVal + " - " + firstVal + '| = ' + range);
        }
```

Maybe it would be clearer to make it a setting

**Comment by hammy2899** [comment]
This setting should be called something like `preventCrossover` and I think it should  be enabled by default since this is unexpected.

**Comment by luckymurari** [comment]
I would like to work on this

**Comment by luckymurari** [comment]
This seems to be too difficult for me :( I give up. Someone else please take it up.

**Comment by ColinFrick** [comment]
I'll look into this.

**Comment by lubber-de** [comment]
Implemented by #1082
